build.sbt

```diff
@@ -635,6 +635,8 @@ lazy val http4s = projectMatrix
         Dependencies.Http4s.dsl.value,
         Dependencies.Http4s.client.value,
         Dependencies.Alloy.core % Test,
+        Dependencies.Alloy.`protocol-tests` % Test,
```

Comment:
do you still need this one if you define it [on L653](https://github.com/disneystreaming/smithy4s/pull/747/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R653)

Reply:
good catch